doc/upgrading.md

```diff
@@ -13,6 +14,31 @@ See the [NEWS](../NEWS.md) summary of changes for each version.
 * [Version 25](#version-25)
 * [Version 20](#version-20)
 
+## Version 30.0
+
+* [RMB-246](https://issues.folio.org/browse/RMB-246) Switch to vertx-pg-client.
+  * Class `SQLConnection` is now provided by RMB. But it was part of
+    old SQL client. `io.vertx.ext.sql.SQLConnection` ->
+    `org.folio.rest.persist.SQLConnection`.
+  * All functions that previusly returned `UpdateResult` now returns
+    `RowSet<Row>`. From that result the number of rows affacted by
+    SQL was `getUpdated()` it is now `rowCount()`.
+  * All functions that previusly returned `ResultSet` now returns
+    `RowSet<Row>`. From that result, the number of rows affacted by
+    SQL is `rowCount()`. The size() method returns number of rows
+    returned. An iterator to go through rows is obtained by calling
+    `iterator`.
```

Comment:
vertx-pg-client returns JSONB as a JsonObject, the old client returned it as a String.

Reply:
Added that, thx